test/modules/TestResultCoordinator/Reports/EdgeHubRestartTest/EdgeHubRestartDirectMethodReport.cs

```diff
@@ -20,7 +21,7 @@ class EdgeHubRestartDirectMethodReport : TestResultReportBase
             ulong senderCount,
             ulong receiverCount,
             TimeSpan medianPeriod)
-            : base(trackingId, resultType)
+            : base(testDescription, trackingId, resultType)
```

Comment:
why do you prefer to put test description as 1st argument, not 3rd one?

Reply:
My logic at the time was that it wasn't worth disturbing the ordering for the metadata, but I felt for the Reports themselves that most useful descriptor for the report should come first. I made this distinction because the report is the presentation, however the report doesn't get serialized in order anyway so this is not sound logic. I should change the reportMetadata for consistency.

Reply:
Just realized I was also inconsistent within the reportMetadata classes as well. I'll lyk when I fix.